plugins/gatsby-plugin-generate-json/gatsby-node.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const path = require('path');
+
+exports.onPostBuild = async ({ graphql, store }, pluginOptions) => {
+  const { program } = store.getState();
+  const { query, serialize } = pluginOptions;
+
+  try {
+    // query graphql
```

Comment:
```suggestion
```

Its fairly obvious that we are querying graphql here, so the comment doesn't seem to really do much.

Reply:
Opps, sorry I left my psuedocode comments in there...Thanks!

Reply:
Oh gotcha! That makes sense 😄